src/domain/session/leftpanel/LeftPanelViewModel.js

```diff
@@ -74,8 +73,14 @@ export class LeftPanelViewModel extends ViewModel {
         return this._settingsUrl;
     }
 
-    get createRoomUrl() { return this._createRoomUrl; }
+    showCreateRoomView() {
+        this.navigation.push("create-room");
+    }
 
+    showJoinRoomView() {
+        this.navigation.push("join-room");
```

Comment:
This is because I forgot to add `join-room` [here](https://github.com/vector-im/hydrogen-web/blob/a10025ddb2a023143d319dd660089f163861947d/src/domain/navigation/index.ts#L21).